src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonParameterInfo.cs

```diff
@@ -28,10 +33,9 @@ internal abstract class JsonParameterInfo
 
         public JsonNumberHandling? NumberHandling { get; private set; }
 
-        // The zero-based position of the parameter in the formal parameter list.
-        public int Position { get; private set; }
+        // Using a field to avoid copy semantics.
```

Comment:
More reasons why it should be a `class`.

Reply:
I presume this concern is no longer applicable? Should it be changed into a property?

Reply:
Thanks, will update in new PR to avoid CI reset.